browser/main/modals/PreferencesModal/SnippetTab.styl

```diff
@@ -127,7 +118,7 @@
     background darken(#f5f5f5, 5)
 
 .snippet-detail
-  width 70%
+  width calc(100% - 33%)
```

Comment:
Why don't you just replace the `calc` with `67%`?

Reply:
@ZeroX-DG You're right. I've replaced it with the static value.